sdcm/remote.py

```diff
@@ -963,3 +967,126 @@ def run_output_check(self, cmd, timeout=None, ignore_status=False,
 
         if not ignore_status and result.exit_status > 0:
             raise process.CmdError(command=result.command, result=result)
+
+
+class RemoteFabric(object):
+
+    # def __init__(self, hostname, user="root", port=22, password="",
+                 # key_file=None, wait_key_installed=0, extra_ssh_options=""):
+    def __init__(self, *args, **kwargs):
+        logger = logging.getLogger('avocado.test')
+        self.log = SDCMAdapter(logger, extra={'prefix': str(self)})
+        self.log.info(args)
+        self.log.info(kwargs)
+        self.key_file = kwargs.get('key_file', "")
+        self.user = kwargs.get('user', "scylla-test")
+        self.port = kwargs.get('port', 22)
+        self.hostname = kwargs.get('hostname', "localhost")
+        self.password = kwargs.get('password', "")
+        self.connect_timeout = kwargs.get('connect_timeout', 60)
+        self.cmd_timeout = kwargs.get('timeout', 300)
+        self.known_hosts_file = tempfile.mkstemp()[1]
```

Comment:
why create a temp file ? why not use:
http://docs.fabfile.org/en/1.14/usage/env.html#disable-known-hosts

Reply:
sorry this one is for older version

Reply:
shame it wasn't ported yet,

I would leave a comment point to here:
https://github.com/fabric/fabric/issues/1804
for us to amend it once bitprophet would decide on it.